naslib/search_spaces/darts/conversions.py

```diff
@@ -254,7 +254,7 @@ def convert_compact_to_genotype(compact):
         genotype.append([])
 
         for j in range(8):
-            genotype[i].append((OPS[cell[j][1] - 1], cell[j][0]))
```

Comment:
@Neonkraft @arberzela  the reason for this is because there is a difference between the op list in (the original darts genotype)[https://github.com/quark0/darts/blob/f276dd346a09ae3160f8e3aca5c7b193fda1da37/cnn/genotypes.py#L5] and the [nasbench301 darts genotype](https://github.com/automl/nasbench301/blob/5df4657d1949e52301067de12a4159b3c8621098/nasbench301/representations.py#L4). The original genotype has 8 ops, including `none` as the 0th op, while the nasbench301 genotype has 7 ops since it does not have `none` .

We used `convert_compact_to_genotype()` for performance predictor experiments, which loaded nasbench301 data. We will need to figure out a solution before running performance predictor experiments on this branch. Maybe we should name the nasbench301 representations something else like nb301_genotype/nb301_compact.